app/src/colorpalettewidget.cpp

```diff
@@ -65,6 +66,25 @@ void ColorPaletteWidget::initUI()
     else
         setGridMode();
 
+    QColor savedColor;
+    savedColor.setRgba(settings.value("colorOfSliders").toUInt());
```

Comment:
@CandyFace Could I ask what "colorOfSliders" is for?

Reply:
Yup, I think it will be more consistent, not having 2 different color values in the system.

Also the front color may change when users load a new project, so the color inspector will need to get the new color when `Editor::objectLoaded()` signal is emitted.

Reply:
Alright I'll change it later today

Reply:
I've added and tested that it works now.

I'm not sure the implementation is the best way to go though. I've added a new connection that is called for the only purpose of loading our color on init. This avoids setColor changing our value back and fourth between values, as it may at some point get a value which it can't parse correctly.

It's the least messy way I can come up with for now, to avoid adding too much logic to setColor.